src/main/java/net/mcreator/ui/dialogs/ListSelectorDialog.java

```diff
@@ -76,8 +83,20 @@ public ListSelectorDialog(MCreator mcreator, Function<Workspace, List<T>> entryP
 		scrollPane.setBorder(BorderFactory.createEmptyBorder(0, 12, 0, 0));
 		mainComponent.add("Center", scrollPane);
 
+		if (customValue) {

```

Comment:
If you add the button depending on `customValue`, I think this condition can be skipped as if `customValue` is false, the button is not added anyways.

Reply:
Done!